themes/src/main/resources/theme/base/admin/resources/js/controllers/realm.js

```diff
@@ -1875,6 +1875,8 @@ module.controller('RealmFlowBindingCtrl', function($scope, flows, Current, Realm
         }
     }
 
+    $scope.enabledFeatures = realm.enabledFeatures;
```

Comment:
Please remove. This information is available on server info (serverInfo.profileInfo.)